jaxrs-api/src/main/java/jakarta/ws/rs/core/Response.java

```diff
@@ -1350,6 +1350,13 @@ public enum Status implements StatusType {
          * @since 2.1
          */
         PRECONDITION_REQUIRED(428, "Precondition Required"),
+        /**
+         * 422 Unprocessable Entity, see <a href="https://datatracker.ietf.org/doc/html/rfc4918#section-11.2">RFC 4918: 
```

Comment:
Please don't refer to WebDAV, but instead go with https://httpwg.org/specs/rfc9110.html#status.422.

Reply:
@mkarg Thanks for the suggestion, I've updated the link